ibis/clickhouse/tests/test_types.py

```diff
@@ -14,3 +14,14 @@ def test_column_types(alltypes):
     assert df.float_col.dtype.name == 'float32'
     assert df.double_col.dtype.name == 'float64'
     assert df.timestamp_col.dtype.name == 'datetime64[ns]'
+
+
+def test_columns_types_with_additional_argument(con):
```

Comment:
It looks like this test is failing: https://circleci.com/gh/ibis-project/ibis/4360?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Yes, it is caused by clickhouse-server version difference. The version I used for testing is 1.1.54385, but docker-compose requires 1.1.54327. https://github.com/yandex/ClickHouse/blob/master/CHANGELOG.md#clickhouse-release-1154337-2018-01-18

Reply:
Should we update the docker image? I'm relatively ignorant about clickhouse, @kszucs is the resident expert on that.

Reply:
IMHO we can just update the clickhouse image, they have frequent releases.

Reply:
I created #1513 to update the image. I'll get that working and then we can merge this PR.